src/components/Common/CrowdsalesList.js

```diff
@@ -29,20 +29,20 @@ export default class CrowdsalesList extends Component {
         <div className="container-fluid">
           <div className="table-row flex-table-header">
             <div className="text">Address</div>
-            <div className="sm-text">Token Name</div>
-            <div className="sm-text">Ticker</div>
-            <div className="sm-text">Date</div>
+            {/*<div className="sm-text">Token Name</div>*/}
```

Comment:
@fernandomg can we delete these commented lines before merging to master?

Reply:
removed